gcr/ansible-vault.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Ensures ansible-vault is installed within the build environment
+#
+# NOTE: This causes an error when running in Travis
```

Comment:
Is this true, or a copy / paste error from a different script?

Reply:
This was based in `ensure-gcloud-installed`. I left the message just in case before testing that, because it's the same principle but I couldn't test that until that gets commited. I'm removing this by now